coordinator/src/orderbook/routes.rs

```diff
@@ -51,6 +53,16 @@ pub async fn get_order(
     Ok(Json(order))
 }
 
+#[instrument(skip_all, err(Debug))]
+pub async fn get_orders(State(state): State<Arc<AppState>>) -> Result<Json<Vec<Order>>, AppError> {
```

Comment:
❓ why is that api needed?

Reply:
I used it to manually retrieve the current orders from the coordinator. I think this endpoint is useful even though it's not being used in the app

Reply:
Gotya, should we move it to admin then and add a comment on why it's there? 🙂